### ISSUE TITLE
Develop docs screenshots issues

### ISSUE THREAD

**Issue by MCGallaspy** [learningequality/ka-lite#4296]
Tried to build the docs with screenshots in order to commit them, on `develop`. Got the error:

```
ERROR:kalite:Error taking screenshot:
ERROR:kalite:Message: $(...).qtip is not a function
```

With maybe more to come once that's resolved. But at any rate we should aim to fix up screenshots a bit during the 0.15 bug bash cycle, at least well enough to build and commit screenshots from _someone's_ machine if not the build server itself.

**Comment by MCGallaspy** [comment]
After getting this working, it would be good to comb through the docs and see if screenshots should be added anywhere. I'm thinking of the learner's user manual specifically.

**Comment by rtibbles** [comment]
Fixed in #4433
